apps/settings/js/panels/operator_settings/model/auto_selection_model.js

```diff
@@ -0,0 +1,82 @@
+define(function() {
+  'use strict';
+
+  var Defer = require('modules/defer');
+  var FeatureModel = require('modules/feature_model');
+  var MobileConnectionWrapper =
+    require('panels/operator_settings/mobile_connection_wrapper');
+
+  // Auto selection state
+  const AS_STATE = {
+    UNKNOWN: 0,
+    ENABLED: 1,
+    DISABLED: 2
+  };
+
+  var AutoSelectionModel = function(connWrapper) {
+    return FeatureModel({
+      onInit: function(onchange) {
+        this._pendingState = null;
```

Comment:
`onchange` is not used in this case because all changes are made completely via the api. I listed it here only for the purpose of showing you that we pass such handler to `onInit`.  We will remove it later.